bql/semantic/expression.go

```diff
@@ -120,7 +127,16 @@ func (e *evaluationNode) Evaluate(r table.Row) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	csEL, csER := formatCell(eL), formatCell(eR)
+
+	csEL, err := formatCell(eL)
+	if err != nil {
+		return false, fmt.Errorf("in evaluationNode.Evaluate got error: %v", err)
```

Comment:
Shouldn't this message say that the error was formatting cell? And maybe which cell it was trying to format? 
Messages like`in evaluationNode.Evaluate` don't add much information.
Please apply the same rationale throughout this file.

Reply:
Done.